src/heapy/stdtypes.c

```diff
@@ -226,7 +226,15 @@ frame_relate(NyHeapRelate *r)
         return 1;
 
     /* stack */
-
+#if PY_MAJOR_VERSION >= 3 && PY_MINOR_VERSION >= 10
+    int i;
+    for (i = 0; i < v->f_stackdepth; i++) {
+        if (v->f_valuestack[i] == r->tgt) {
+            if (r->visit(NYHR_STACK, PyLong_FromSsize_t((&v->f_valuestack[i])-v->f_valuestack), r))
```

Comment:
If I understand this correctly, this changed f_valuestack from a linked list to an array right? Could you add a temporary variable `p` here for `&v->f_valuestack[i]` so the expression is more easily readable?

Reply:
Ah I see.